lib/src/10_auth/auth_screen.dart

```diff
@@ -128,10 +146,45 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
     );
   }
 
-  Widget _buildCreateAccountScreen(BuildContext context) {
-    final TextEditingController emailController =
-        new TextEditingController(text: this._email);
+  Widget _buildTextBox(
+      TextEditingController controllerName,
+      FocusNode currentNode,
+      FocusNode nextNode,
+      String fieldName,
+      bool autoFocus,
+      Future<Widget> Function() submit) {
+    handleKey(RawKeyEvent key) {
+      if (key is RawKeyDownEvent) {
+        if (key.physicalKey.debugName == 'Enter') {
+          submit();
+        } else {
+          {
+            setState(() {
+              this._error = "";
```

Comment:
You can replace it with shorter statement:
setState(() => this._error = "");

Reply:
done